tests/BuildScriptGeneratorCli.Tests/BuildpackDetectCommandTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Oryx.BuildScriptGeneratorCli;
+using System;
+using System.IO;
+using Xunit;
+using Microsoft.Oryx.Tests.Common;
+using Microsoft.Oryx.BuildScriptGenerator.Node;
+using Microsoft.Oryx.Common;
+using Microsoft.Oryx.BuildScriptGenerator;
+
+namespace BuildScriptGeneratorCli.Tests
+{
+    public class BuildpackDetectCommandTest : IClassFixture<TestTempDirTestFixture>
+    {
+        private readonly string _testDirPath;
+
+        public BuildpackDetectCommandTest(TestTempDirTestFixture testFixture)
+        {
+            _testDirPath = testFixture.RootDirPath;
+        }
+
+        [Fact]
+        public void Execute_OutputsNode_WhenPackageJsonExists()
+        {
+            // Arrange
+            var cmd = new BuildpackDetectCommand { PlatformDir = string.Empty, PlanPath = string.Empty };
+            var console = new TestConsole();
+            var svcProvider = GetServiceProvider(cmd);
+
+            // Act
+            int exitCode;
+            using (new CurrentDirectoryChange(_testDirPath))
```

Comment:
I am just wondering if this would have wider impact...like tests in different classes run in parallel and if this would cause  breaks which are subtle...is there any other way to test this? Should/is this command always supposed to run from current directory only?